### ISSUE TITLE
If you assign a tag to a resource group, all the Azure resources in that resource group are assigned to the same tag.

### ISSUE THREAD

**Issue by EChDatAnyl** [Ditectrev/Microsoft-Azure-AZ-900-Microsoft-Azure-Fundamentals-Exam-Questions-Answers#7]
https://learn.microsoft.com/en-us/training/modules/describe-cost-management-azure/7-describe-purpose-of-tags

This is a Yes in Section An example tagging structure: A resource tag consists of a name and a value. **You can assign one or more tags to each Azure resource.**

**Comment by danieldanielecki** [comment]
Yes, and also because when applying tags to a resource group the resource in the resource group don't inherit those tags.
